website/blog/2022-07-28-srb-tapioca.md

```diff
@@ -0,0 +1,266 @@
+---
+id: srb-tapioca
+title: Tapioca is the recommended way to generate RBIs for Sorbet
+---
+
+Today we're excited to announce that the Sorbet team officially recommends using
+Shopify's [Tapioca](https://github.com/Shopify/tapioca) gem for RBI generation
+in projects that use Sorbet. Tapioca takes the place of the `srb rbi` family of
+commands, which as of today are officially in maintenance mode. Sorbet itself
+remains in active development.
+
+<!--truncate-->
+
+![sorbet plus tapioca](/img/sorbet-plus-tapioca.svg)
+
+To elaborate, here's what's happening:
+
+- The [`tapioca`](https://github.com/Shopify/tapioca) gem is now the recommended
+  approach to [RBI generation](/docs/rbi) in projects that use Sorbet. This
+  recommendation is reflected in the Sorbet docs. Tapioca replaces (and improves
+  on!) functionality currently provided by the `srb rbi` command. Tapioca is
+  maintained by Shopify and powers RBI generation in all of their Ruby
+  codebases. We'll discuss Tapioca in more depth below.
+
+- We're putting `srb rbi` (and by extension, certain phases of `srb init`) into
+  maintenance mode. This largely codifies what was already the case. It will
+  remain available for the foreseeable future, but may be removed at some point
+  if it breaks in a way that would require substantial effort to repair.
+
+- Stripe remains actively invested in developing Sorbet itself (aka, `srb tc`)
+  as well as the runtime support libraries (`sorbet-runtime`). In fact, 2022 has
+  been
+  [one of the most active periods](https://github.com/sorbet/sorbet/graphs/contributors?from=2022-01-01&to=2022-07-26&type=c)
+  in Sorbet's history, with nearly 700 commits from 30 different contributors so
+  far this year.
+
+As of today Sorbet's guide to
+[Adopting Sorbet in an Existing Codebase](/docs/adopting) has been updated to
+recommend using Tapioca. The old guide remains available in the
+[repo history](https://github.com/sorbet/sorbet/blob/834c45a98cc52c2474b75fc58d06d4526888f48f/website/docs/adopting.md)
+if you need to reference it.
+
+If you'd like to start using Tapioca in a project that already uses `srb rbi`,
+[see this migration guide](#TODO).
+
+With the announcements out of the way, let's dive into some questions we imagine
+you might have.
+
+## What is Tapioca?
+
+[Tapioca](https://github.com/Shopify/tapioca) is a Ruby gem that provides
+command line tools for generating [RBI files](/docs/rbi). From its
+documentation, "Tapioca surfaces types and methods from many sources that Sorbet
+cannot otherwise see—such as gems, Rails and other DSLs—compiles them into RBI
```

Comment:
```suggestion
cannot otherwise see — such as gems, Rails and other DSLs — compiles them into RBI
```

Reply:
It doesn't render well, but these are em dashes:

<img width="806" alt="image" src="https://user-images.githubusercontent.com/5544532/181310498-38272e99-2b28-436e-8258-a86c36d56a3b.png">

which I prefer to use to typeset dashed, parenthetical expressions like this.